### PR TITLE
Add cli option for limit-per-account in the mempool

### PR DESCRIPTION
**Pull request by MakisChristou**
This PR makes the already existing flags 

- `txpool-limit-per-account` (default 16)
- `txpool-limit` (default 10k)

which were previously solo only flags available in testnet and mainnet modes. No unbounded option was added but we can discuss if that is necessary.

### CODE REVIEW

File: cmd/thor/flags.go [hunk 1]
```diff
@@ -150,6 +150,16 @@ var (
 		Value: "localhost:2112",
 		Usage: "metrics service listening address",
 	}
+	txPoolLimitFlag = cli.Uint64Flag{
```
Comment (leszek-vechain) [inline cmd/thor/flags.go:153]: do we need to update docs as it not only solo flag now

File: cmd/thor/main.go [hunk 2]
```diff
@@ -93,6 +93,8 @@ func main() {
 			disablePrunerFlag,
 			enableMetricsFlag,
 			metricsAddrFlag,
+			txPoolLimitFlag,
```
Comment (libotony) [inline cmd/thor/main.go:96]: Txpool limit should be a solo-only flag.

**Review by otherview — APPROVED** [review]
lgtm 👍 

Do you think it is worth creating a test for the limit per account ?

**Comment by leszek-vechain** [comment]
would be nice to have for sure, just to ensure that this change works as expected

**Comment by MakisChristou** [comment]
> lgtm 👍
> 
> Do you think it is worth creating a test for the limit per account ?

Do you mean e2e tests or unit tests. Because there are already some unit tests for the account limit.

**Comment by otherview** [comment]
> > lgtm 👍
> > Do you think it is worth creating a test for the limit per account ?
> 
> Do you mean e2e tests or unit tests. Because there are already some unit tests for the account limit.

Yeah, It guess that's fine, not worth the whole setup for the tx per account test. 
Good stuff !

**Pull request merged**